lib/DDG/Spice/Shares.pm

```diff
@@ -0,0 +1,20 @@
+package DDG::Spice::Shares;
+# ABSTRACT: Stock quote ticker
+
+use strict;
+use DDG::Spice;
+
+triggers startend => 'stock quote for', 'stock quote', 'stock price of', 'stock price', 'share price', 'stocks', 'shares', 'share price for', 'share price of';
+
+spice to => 'https://og6c1r82zb.execute-api.us-east-1.amazonaws.com/v1/ddg?term=$1';
```

Comment:
@stratmm thanks for clarifying. This should be fine, I just wanted to confirm, this wasn't something unofficial or illegitimate 👍